server/routes/shared/layoutPresenter.ts

```diff
@@ -1 +1,8 @@
-export default class LayoutPresenter {}
+import { DeliusServiceUser } from '../../services/communityApiService'
+import ServiceUserBannerPresenter from './serviceUserBannerPresenter'
+
+export default class LayoutPresenter {
+  constructor(private readonly serviceUser: DeliusServiceUser | null) {}
```

Comment:
Have you considered making this a default argument, so we don't have to add in all the `null`s everywhere?

If so, this could probably come as a refactor when we start adding the service user banner in more places, as it'll be easier to replace the `null`s.

Reply:
Yeah, I did consider it and then didn’t, for some reason. It’d probably be a good idea, but as you say I think it can wait.